docs/modules/ROOT/pages/how-tos/change-parameter.adoc

```diff
@@ -0,0 +1,37 @@
+= Change a Parameter
+
+Suppose you want to overwrite an inventory parameter for a cluster or tenant.
+This page assumes that you have already set up access to Lieutenant, so that `commodore catalog compile` generally works from any directory.
```

Comment:
Probably want to add a note about a local installation of `commodore` somewhere in `$PATH` or an alias running dockerized Commodore, if running `commodore catalog compile` should work from any directory.